### PR TITLE
DB queries for getting audit history by acronym

### PR DESCRIPTION
**Pull request by loneil**
# Description
Populate DB call for getting audit history.
This schema will change with new history setup after UX decisions, so not permanent, but structure in place now.

## Types of changes

New feature (non-breaking change which adds functionality) 

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-k8vopl-tools.pathfinder.gov.bc.ca/dashboard?id=nr-get-token-pr-116

File: backend/src/components/audit.js [hunk 1]
```diff
@@ -18,17 +18,25 @@ const audit = {
...
       if (history) {
-        return history;
+        const hsts = history.Lifecycles.map(lc => {
+          const hst = {};
+          hst.date = lc.createdAt;
+          hst.acronym = history.acronym;
+          hst.user = lc.LifecycleHistories[0].User.username;
+          hst.environment = lc.env;
+          hst.details = lc.appConfig;
+          return hst;
```
Comment (jujaga) [inline backend/src/components/audit.js]: As this is direct object assignments, it'd be better to write it as the following:
```
return {
  acronym: history.acronym,
  date: lc.createdAt,
  details: lc.appConfig,
  environment: lc.env,
  user: lc.LifecycleHistories[0].User.username
}
```
This is slightly more memory efficient as there's one less object constant being instantiated -> instead the object is just constructed as part of the anonymous function definition directly.

File: backend/src/services/audit.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+const db = require('../models');
+
+module.exports = {
+  async findHistory(acronym) {
+    return await db.Acronym.findOne({
+      where: { acronym: acronym },
```
Comment (jujaga) [inline backend/src/services/audit.js]: Might not be a bad idea to have this where be after the include attribute (alphabetical ordering convention seen in other models)

File: backend/src/components/audit.js [hunk 3]
```diff
@@ -1,8 +1,8 @@
 const log = require('npmlog');
 
-// const {
-//   auditService
-// } = require('../services');
+const {
+  auditService
+} = require('../services');
```
[the remark this answers is not in the document]
Comment (loneil) [reply backend/src/components/audit.js]: Fixed

File: backend/src/components/audit.js (hunk 1 above)
Comment (loneil) [reply backend/src/components/audit.js]: Right, good call. Fixed

File: backend/src/services/audit.js (hunk 2 above)
Comment (loneil) [reply backend/src/services/audit.js]: Yes, fixed

**Pull request merged**